src/wp-includes/template.php

```diff
@@ -699,16 +700,19 @@ function get_attachment_template() {
  * @return string The template filename if one is located.
  */
 function locate_template( $template_names, $load = false, $load_once = true, $args = array() ) {
+	$stylesheet_path = get_stylesheet_directory();
+	$template_path   = get_template_directory();
+
```

Comment:
```suggestion
     $is_child_theme = $template_path !== $stylesheet_path;
```

Reply:
Addressed in https://github.com/WordPress/wordpress-develop/pull/5160/commits/928936f22dcf2bb0c16e98aaa910ce65dfba1780